packages/mint-components/CHANGELOG.md

```diff
@@ -7,10 +7,37 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [1.6.8] - 2023-04-20
+## [1.6.8] - 2023-5-16
 
 ### Changed
+
```

Comment:
Changelog likely needs some review, from a glance it seems some things are missing. For example the sqm-close-button, might be more